tox.ini

```diff
@@ -116,9 +116,9 @@ envlist =
     ; opentelemetry-instrumentation-flask
     ; The numbers at the end of the environment names
     ; below mean these dependencies are being used:
-    ; 0: Flask ==2.1.3 Werkzeug <3.0.0
-    ; 1: Flask ==2.2.0 Werkzeug <3.0.0
-    ; 2: Flask >=3.0.0 Werkzeug >=3.0.0
+    ; 0: Flask==2.1.3 Werkzeug==2.3.8
+    ; 1: Flask==2.2.0 Werkzeug==2.3.8
+    ; 2: Flask==3.0.2 Werkzeug==3.0.1
```

Comment:
I'm a bit confused what's going on here. This version constraint is impossible to fulfill

How did we get here? 😅